.vscode/settings.json

```diff
@@ -83,6 +83,26 @@
     "xlocmon": "cpp",
     "xloctime": "cpp",
     "xtree": "cpp",
-    "*.inc": "cpp"
+    "*.inc": "cpp",

```

Comment:
So far I've just been using the default `.gitignore` file provided by WPILib. I agree these merge conflicts are annoying; I'll have it ignore everything in the `.vscode` folder (we don't use `.idea`) and if it creates problems we can change it back.

Reply:
Where did you get this `.gitignore` from? It's a bit weird because some files in `.vscode` are ignored and some aren't:
```
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
```
The `.gitignore` of wpilib here ignores the whole `.vscode` directory: [https://github.com/wpilibsuite/vscode-wpilib/blob/main/.gitignore](https://github.com/wpilibsuite/vscode-wpilib/blob/main/.gitignore)

Reply:
It's auto generated when a new WPILib vscode project is made. Pretty sure this is the official file: https://github.com/wpilibsuite/vscode-wpilib/blob/main/vscode-wpilib/resources/gradle/shared/.gitignore

Reply:
Okay, this seems fine. I'll create a better way to solve this and create PR for it this week.